pkg/app/AppCrudOperationService.go

```diff
@@ -457,13 +458,77 @@ func convertUrlToHttpsIfSshType(url string) string {
 	return httpsURL
 }
 
+// updateDuplicatedEntriesInApp returns err after the marking the duplicated apps entry false on the basis of its creation time...
+func (impl AppCrudOperationServiceImpl) updateDuplicatedEntriesInApp(earlyApp *appRepository.App, duplicatedApp *appRepository.App, installedApp repository2.InstalledApps) error {
+	return impl.installedAppDbService.UpdateDuplicatedEntriesInAppAndInstalledApps(earlyApp, duplicatedApp, &installedApp)
+}
+
+// handleDuplicateAppEntries identifies and resolves duplicate app entries based on creation time.
+// It marks the most recent duplicate entry as inactive and updates the corresponding installed app.
+func (impl AppCrudOperationServiceImpl) handleDuplicateAppEntries(appNameUniqueIdentifier string) (*appRepository.App, error) {
+	// Fetch app IDs by name
+	appIds, err := impl.getAppIdsByName(appNameUniqueIdentifier)
+	if err != nil {
+		impl.logger.Errorw("error in fetching app Ids by appName", "appNameUniqueIdentifier", appNameUniqueIdentifier, "err", err)
```

Comment:
error in fetching app Ids by appName - > error in fetching app Ids by appIdentifier